notifications_utils/letter_timings.py

```diff
@@ -12,6 +13,15 @@
 ]
 
 
+bank_holidays_monday_to_friday = BankHolidays(
```

Comment:
I wonder if it's better to call this `working_days_monday_to_friday` or `working_days_printers` or `working_days_dvla`